src/desktop/components/main_layout/header/templates/minimal_header.jade

```diff
@@ -1,3 +1,5 @@
 nav#main-layout-minimal-header
-  a.logo( href='#{ headerLogoHref || "/" }')
-    include ../../public/icons/logotype.svg
+  .logo
+    unless options.hideLogo
```

Comment:
Just out of curiosity, was there a reason to pull out `.logo` as you did?

Instead of just adding the conditional:
```jade
unless options.hideLogo
  a.logo( href='#{ headerLogoHref || "/" }')
    include ../../public/icons/logotype.svg
```

Reply:
Yup, it's the `svg` tag within the `a.logo` block in the CSS that gave the header its height. Without moving this, the logo would be hidden but the browser would move the content of the page up, and we want to maintain the top margin.